.sqlx/query-b028521a1d2efb7113cd45e88d0118b38e58aaa900930e6de4c9d56dae80bc05.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
I guess this file should not be here? Probably should exclude `.sqlx` in gitignore?

Reply:
This file is actually supposed to be there. sqlx does compile time checking against a live database, but when we distribute this as a library, obviously the user does not have a database handy and sqlx should not give errors about that. You can use `cargo sqlx prepare` to prepare this file and it will stop complaining to users of the library.